packages/atomic/cypress/integration-insight-panel/insight-panel.cypress.ts

```diff
@@ -67,6 +67,15 @@ describe('Insight Panel test suites', () => {
         .should('have.attr', 'href');
     });
 
+    it('should display result action bar', () => {
+      InsightPanelsSelectors.resultActionBar()
```

Comment:
Would we be able to trigger the hover and assert that the buttons are actually visible?

Reply:
ah i see thanks